CHANGELOG.md

```diff
@@ -18,12 +20,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rewrote "cannot find field for current simulation time" error message for clarity
 - Updated `HCO_Error` and `HCO_Warning` to call `HCO_WordWrapPrint` to wrap messages to 78 characters wide
 - Replaced terse error messages in `hcoio_read_std_mod.F90` with more descriptive ones
+- Updated error messages in routine `ExtStateInit` to be more descriptive
+- Changed call to `ExtDat_Set` for `ExtData%SNOWHGT` to read data from the GMAO `SNOMAS` field instead of `SNOWHGT`
+- Updated `run/HEMCO_sa_Spec.rc` to be consistent with the new dust species
+- Updated `run/createRunDir.sh` to copy the`cleanRunDir.sh` script to the run directory
+- Updated comment headers and removed GCHP-specific text in `run/cleanRunDir.sh`
+- Updated `DustL23M` extension to read the `C_tune` parameter from the HEMCO configuration file
```

Comment:
I don't think we need this comment because this is a new extension introduced in this PR.

Reply:
This has now been removed.